spec/models/concerns/hyrax/ability_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+require 'cancan/matchers'
+
+RSpec.describe Hyrax::Ability do
+  subject(:ability) { ability_class.new(user) }
+  let(:user) { FactoryBot.create(:user) }
+
+  shared_context 'with guest user' do
+    let(:user) { FactoryBot.create(:user, :guest) }
+  end
+
+  shared_context 'with deposit access on an admin set' do
+    let(:permission_template) { FactoryBot.create(:permission_template, with_admin_set: true) }
+
+    before do
+      FactoryBot.create(:permission_template_access,
+                        :deposit,
+                        permission_template: permission_template,
+                        agent_type: 'user',
+                        agent_id: user.user_key)
+    end
+  end
+
+  shared_context 'with create access on a work type' do
+    let(:work_model) { Monograph }
+
+    before { ability.can(:create, work_model) }
+  end
+
+  let(:ability_class) do
+    module Hyrax
+      module Test
+        module AbilityMixin
+          class Ability
+            include Blacklight::AccessControls::Ability
```

Comment:
Should this be `Hydra::Ability` to match the install generator and the example in Hyrax::Ability?

Reply:
i'm working these tests up from scratch, only adding functionality needed to pass them.

part of my goal is to figure out whether `Hydra::Ability` is a hard dependency for this mixin, or whether it's just most often included.